### PR TITLE
Gornovanton1992 eh1 t1

### PR DESCRIPTION
**Pull request by gornovanton1992**

### CODE REVIEW

**Review by GardDen — APPROVED** [review]
Lgfm

File: homework/english/gornovanton1992/EH1t1.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+#
+GRAMMAR PRACTICE
+TASK1
+CHOOSE THE RIGHT OPTION (ВЫБЕРИТЕ ПРАВИЛЬНЫЙ ВАРИАНТ).
+1 He is usually having] breakfast at 6 a.m.
```
Comment (DenisAbramovDm) [inline homework/english/gornovanton1992/EH1t1.md:5]: Has

File: homework/english/gornovanton1992/EH1t1.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+2 Anyone who smokes 20 cigarettes a day can expect to have health problems.
+3 She doesn't like cooking, but today she [bakes/is baking] a pie.
```
Comment (DenisAbramovDm) [inline homework/english/gornovanton1992/EH1t1.md:7]: Is baking

**Review by DenisAbramovDm — APPROVED** [review]
Lgfm

**Review by VasiliZas — APPROVED** [review]

**Pull request merged**